### DISCUSSION TITLE
Linking out to external resources in dp.HTML()

### DISCUSSION THREAD

**Discussion by ArtemSokolov** [datapane/datapane#369] [Ideas]
Dear Datapane developers,

The documentation notes that

> The HTML block is sandboxed and cannot execute JavaScript.

However, this seems to prevent reports from linking out to external pages in `dp.HTML()`. For example,

```
html = '<a target="_blank" href="https://datapane.com/">Created in the awesome Datapane</a>'
dp.App(dp.HTML(html)).save('test.html')
```

generates a page where the link does not appear to be clickable. Is there a way to get the desired behavior from `dp.HTML()`?

**Comment by mands** [comment]
Hey @ArtemSokolov, that should work, pinging @Buroni who is more familiar with the HTML block implementation in the browser.

Just to note that we are modifying the security policies shortly that should allow more extensive JS within the HTML block going forwards.

**Comment by Buroni** [comment]
Hey @ArtemSokolov we use sandboxed iframes which block the iframe document from navigating away. I'll look into allowing link navigation and get back to you on this in the next few days.

**Comment by ArtemSokolov** [comment]
Thanks, @Buroni. No rush. Our current workaround has been to right-click -> open in a new tab.
